.betterer.ts

```diff
@@ -1,5 +1,17 @@
 import { regexp } from '@betterer/regexp';
+import { eslint } from '@betterer/eslint';
 
 export default {
-  'no enzyme tests': () => regexp(/from 'enzyme'/g).include('**/*.test.*')
+  'no enzyme tests': () => regexp(/from 'enzyme'/g).include('**/*.test.*'),
+  'no type assertions': () => eslint({
+    '@typescript-eslint/consistent-type-assertions': [
+      'error',
+      {
+        assertionStyle: 'never',
+      },
+    ],
+  }).include('**/*.ts*'),
+  'no explicit any': () => eslint({
```

Comment:
Maybe I'm reading this wrong but does this run eslint twice across the entire codebase? Would grouping these two tests together speed things up?